.github/workflows/diff.yaml

```diff
@@ -15,496 +15,10 @@ on:
       - "licenses/*"
 
 jobs:
-  community_build:
```

Comment:
Instead of deleting everything but jepsen from `diff` for testing, you can just add
`if: false` in the same level as `name: `. The effect will be the same

Reply:
This is a tip for future work

Reply:
Resolve this after reading :+1: